test_empty_list (test_one_hot_encoding_unit.TestOneHotEncoder) ... FAIL
test_one_word (test_one_hot_encoding_unit.TestOneHotEncoder) ... ok

======================================================================
FAIL: test_empty_list (test_one_hot_encoding_unit.TestOneHotEncoder)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/akimkalenuk/Downloads/HW_Test_Python_AAA/issue_3/test_one_hot_encoding_unit.py", line 29, in test_empty_list
    fit_transform(cities)
AssertionError: TypeError not raised

----------------------------------------------------------------------
Ran 2 tests in 0.001s

FAILED (failures=1)
